<launch>
    <arg name="angle_thres" default="80" doc="Angle threshold value"/>
    <arg name="range_thres" default="0.30" doc="Range threshold value"/>
    <arg name="intensity_thres" default="5" doc="Intensity threshold value"/>

  <!-- launch velocity_estimator_node -->
    <node pkg="radar_velocity_estimator" type="velocity_estimator_node.py" name="dopplerLogger" output="screen">
        <param name="angle_thres" value="$(arg angle_thres)"/>
        <param name="range_thres" value="$(arg range_thres)"/>
        <param name="intensity_thres" value="$(arg intensity_thres)"/>
    </node>

    <!-- launch inertial2BodyFrame Velocity transform node -->
    <node pkg="static_tfs" type="vrpn2BodyFrameVelocity.py" name="vrpn2BodyFrameVelocity_node" output="screen"/>

</launch>
